alpakaConfig.cmake

```diff
@@ -448,6 +463,147 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
+          STRING(COMPARE EQUAL "${ALPAKA_HIP_ARCH}" "sm_13" IS_HIP_ARCH_UNSUPPORTED)
+
+          IF(IS_HIP_ARCH_UNSUPPORTED)
+              MESSAGE(WARNING "Unsupported HIP architecture ${ALPAKA_HIP_ARCH} specified. SM 2.0 or higher is required for HIP/CUDA 7.0. Using sm_20 instead.")
+              SET(ALPAKA_HIP_ARCH sm_20 CACHE STRING "Set GPU architecture" FORCE)
+          ENDIF(IS_HIP_ARCH_UNSUPPORTED)
+
+          SET(ALPAKA_HIP_COMPILER "hipcc" CACHE STRING "HIP compiler")
+          SET_PROPERTY(CACHE ALPAKA_HIP_COMPILER PROPERTY STRINGS "hipcc")
+          # the ;clang was removed from the property value list for the above function
+          OPTION(ALPAKA_HIP_FAST_MATH "Enable fast-math" ON)
+          OPTION(ALPAKA_HIP_FTZ "Set flush to zero for GPU" OFF)
+          OPTION(ALPAKA_HIP_SHOW_REGISTER "Show kernel registers and create PTX" OFF)
+          OPTION(ALPAKA_HIP_KEEP_FILES "Keep all intermediate files that are generated during internal compilation steps (folder: nvcc_tmp)" OFF)
+
+#------------------------
+#no clang support for HIP. not today.
+#------------------------
+#            IF(ALPAKA_CUDA_COMPILER MATCHES "clang")
+#                LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "--cuda-gpu-arch=${ALPAKA_CUDA_ARCH}")
+#
+#                # This flag silences the warning produced by the Dummy.cpp files:
+#                # clang: warning: argument unused during compilation: '--cuda-gpu-arch=sm_XX'
+#                # This seems to be a false positive as all flags are 'unused' for an empty file.
+#                LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-Qunused-arguments")
+#
+#                # Silences warnings that are produced by boost because clang is not correctly identified.
+#                LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-Wno-unused-local-typedef")
+
+#                IF(ALPAKA_CUDA_FAST_MATH)
+#                    # -ffp-contract=fast enables the usage of FMA
+#                   LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-ffast-math" "-ffp-contract=fast")
+#               ENDIF()
+#
+#                IF(ALPAKA_CUDA_FTZ)
+#                    LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-fcuda-flush-denormals-to-zero")
+#               ENDIF()
+
+#             IF(ALPAKA_CUDA_SHOW_REGISTER)
+#                  LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-Xcuda-ptxas=-v")
+#               ENDIF()
+#
+#              IF(ALPAKA_CUDA_KEEP_FILES)
+#                   LIST(APPEND _ALPAKA_COMPILE_OPTIONS_PUBLIC "-save-temps")
+#                ENDIF()
+
+#            ELSE()
+
+                IF(CUDA_VERSION VERSION_EQUAL 8.0)
+                    LIST(APPEND HIP_HIPCC_FLAGS "-Wno-deprecated-gpu-targets")
+                ENDIF()
+
+                IF(NOT CUDA_VERSION VERSION_LESS 7.5)
+                    LIST(APPEND HIP_HIPCC_FLAGS "--expt-extended-lambda")
+                ENDIF()
+
+                LIST(APPEND HIP_HIPCC_FLAGS "-arch=${ALPAKA_HIP_ARCH}")
+
+                IF(NOT MSVC)
```

Comment:
Have you ever tried using it with MSVC? Else we may simply disable the backend here as well.